Code/Tools/ProjectManager/Source/ProjectExportWorker.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <ProjectInfo.h>
+#include <AzCore/Outcome/Outcome.h>
+
+#include <QObject>
+#include <QProcessEnvironment>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QProcess)
+
+namespace O3DE::ProjectManager
+{
+    class ProjectExportWorker : public QObject
+    {
+        // QProcess::waitForFinished uses -1 to indicate that the process should not timeout
+        static constexpr int MaxExportTimeMSecs = -1;
+        inline static const char * ExportCancelled =   "Export Cancelled.";
+        inline static const char * LogOpenFailureMsg = "Failed to open log file.";
+        inline static const char * LogPathFailureMsg = "Failed to retrieve log file path.";
```

Comment:
`inline` shouldn't be needed if these constants are outside of the class. You can just move the constants outside of the class and just use the namespace.  (Look at this [example](https://github.com/o3de/o3de/blob/14f128b6319eec8e8ecb9ca4fbaa6fdf821585d6/Code/Editor/Plugins/ProjectSettingsTool/PlatformSettings_common.h#L41-L58)